copy-fixtures.js

```diff
@@ -0,0 +1,31 @@
+import { promisify } from "node:util";
+import chip from "node:child_process";
+import fs from "node:fs";
+import path from "node:path";
+
+const exec = promisify( chip.exec );
+
+await exec( "npm install --no-save jquery" );
+
+const fixtures = path.join( ".", "test", "fixtures" );
+
+try {
+	await fs.promises.mkdir( path.join( ".", "test" ) );
+	await fs.promises.mkdir( fixtures );
+} catch ( _ ) {
+	await fs.promises.rm(
+		fixtures,
+		{
+			recursive: true
+		}
+	);
+
+}
+
+await fs.promises.cp(
+    path.join( ".", "node_modules", "jquery", "src" ),
```

Comment:
Isn't copying the built file enough?

Reply:
The tests are for the actual source code, not the built file. I saw `TODO: do not use the built version to check the code` in the code here, and I agreed it is better to test the real source code rather than the built file. That way, you don't have to add extra ignoring of max-len and so forth.

Reply:
OK, fair enough.